snippets/submit_payment.py

```diff
@@ -15,24 +16,24 @@
 client = JsonRpcClient("https://s.altnet.rippletest.net:51234")
 
 # Creating two wallets to send money between
-wallet1 = generate_faucet_wallet(client, debug=True)
-wallet2 = generate_faucet_wallet(client, debug=True)
+wallet = generate_faucet_wallet(client, debug=True)
+destination = "rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe"
```

Comment:
If news users want to debug an issue (or) explore transactions on the testnet, they will see a flood of transactions on this faucet-destination account. This might not be intuitive user experience.

Perhaps a helpful comment noting that this is the faucet account might be instructive here.